bin/v-create-dns-cluster-config

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+# info: creates local and remote transfer keys for bind.
+# options: HOST IP
+#
+# This function creates a new transfer key for the host
+# provided and sends it to the new host
+
+#----------------------------------------------------------#
+#                Variables & Functions                     #
+#----------------------------------------------------------#
+
+# Argument definition
+host=$1
+ip=$2
+
+# Includes
+# shellcheck source=/etc/hestiacp/hestia.conf
+source /etc/hestiacp/hestia.conf
+# shellcheck source=/usr/local/hestia/func/main.sh
+source $HESTIA/func/main.sh
+# load config file
+source_conf "$HESTIA/conf/hestia.conf"
+
+#----------------------------------------------------------#
+#                    Verifications                         #
+#----------------------------------------------------------#
+
+check_args '2' "$#" 'HOST IP'
+is_system_enabled "$DNS_SYSTEM" 'DNS_SYSTEM'
+
+# Perform verification if read-only mode is enabled
+check_hestia_demo_mode
+
+#check if openssl is installed
+openssl_installed=$(which openssl | grep openssl)
+
+#----------------------------------------------------------#
+#                       Action                             #
+#----------------------------------------------------------#
+
+# Get Transfer-Key File
+if [ -e '/etc/bind/transferkeys.conf' ]; then
+	transfer_keys='/etc/bind/transferkeys.conf'
+fi
+# Get DNS Config file
+if [ -e '/etc/bind/named.conf' ]; then
+	dns_conf='/etc/bind/named.conf'
+fi
+
+# create new transfer-key
+hostkey=$(openssl rand -base64 64)
+
+# Insert values into transferkeys file
+transkey="key \"$host\" {"
+transkey="transkey \algorithm hmac-sha384;"
```

Comment:
Should be transkey="$transkey .... instead also unless \ needs to be there should probably \n instead for a new line but we should probally keep it on one line any way

Reply:
I've rewritten the whole thing, it didn't do what it should have anyway. 
For readability, it will now look like this: 
key "hast1.example.com" {
        transkey algorithm hmac-sha384;
        transkey secret "ccRjlbZHw8d0eYq0C66vkCbmiQtB2Bgg02+WrNl7u7xM9jLn3Dq8MqVSwZDx5CE/KBOLY5FR6i+4mU7YLkcYvA==";
};